cluster/ubuntu/util.sh

```diff
@@ -231,6 +231,11 @@ EOF
 }
 
 function create-kubelet-opts() {
```

Comment:
This is a slight maintainence headache. Can you move this function into hack/lib and re-use it from local-up-cluster? That would kill 2 birds with 1 stone, and people who invoke local-up (>>number of people who invoke ubuntu setup) will be aware when it's broken.